starburst-presto/presto.sh

```diff
@@ -123,23 +122,47 @@ function configure_hive(){
 connector.name=hive-hadoop2
 hive.metastore.uri=${metastore_uri}
 EOF
+
+# Add new connectors configs here 
+# For example: Adding jmx properties like
```

Comment:
It says about "jmx", but the next line is about "tpch". Please remove this line. 
```suggestion
```